content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,308 @@
+---
+title: API Server Flow Control
+content_template: templates/concept
+min-kubernetes-server-version: v1.18
+---
+
+{{% capture overview %}}
+
+Controlling the behavior of the Kubernetes API server in an overload situation
+is a key task for cluster administrators. The{{< glossary_tooltip term_id="kube-apiserver" text="kube-apiserver" >}} has some controls
+available (i.e. the `--max-requests-inflight` and
+`--max-mutating-requests-inflight` command-line flags) to limit the amount of
+outstanding work that will be accepted, preventing a flood of inbound requests
+from overloading and potentially crashing the API server, but beyond that, the
+Priority and Fairness (or flow-control) API provides a configurable mechanism to
+ensure that in a high-load situation, forward progress will be made on the right
+requests. It also allows for a limited amount of queuing, so that in the event
+of _bursty_ traffic that remains below acceptable levels on average, no requests
+need be dropped.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Enabling the Flow Control API
+
+{{< feature-state state="alpha"  for_k8s_version="v1.18" >}}
+
+The Flow Control API is part of the API Priority and Fairness
+[feature](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md),
+which is not enabled by default. To enable it, add the following command-line flags
+to your `kube-apiserver` invocation:
+
+```shell
+kube-apiserver \
+--feature-gates=APIPriorityAndFairness=true \
+--runtime-config=flowcontrol.apiserver.k8s.io/v1alpha1=true \
+ # …and other flags as usual
+```
+
+## Concepts
+There are several distinct features involved in the flow control API. Incoming
+requests are classified by attributes of the request using _FlowSchemas_, and
+assigned to priority levels. Priority levels add a degree of isolation by
+maintaining separate concurrency limits, so that requests assigned to different
+priority levels cannot starve each other. Within a priority level, a
+fair-queuing algorithm prevents requests from different sources from starving
+each other, and allows for requests to be queued to prevent bursty traffic from
+causing failed requests when the average load is acceptably low.
+
+### Priority Levels
+Without the Priority and Fairness feature enabled, overall concurrency in the
+API server is limited by the `kube-apiserver` flags `--max-requests-inflight`
+and `--max-mutating-requests-inflight`. With the feature enbaled, the
+concurrency limit defined by these flags is divided up among a configurable set
+of _priority levels_. Incoming requests are assigned to a single priority level,
```

Comment:
"Incoming requests are assigned to a single priority level" mixes pluralities.  I would say something more like "Each incoming request is assigned to a single priority level".